cardano-cli/test/cardano-cli-golden/files/golden/help/experimental_governance_committee_create-cold-key-resignation-certificate.cli

```diff
@@ -0,0 +1,16 @@
+Usage: cardano-cli experimental governance committee create-cold-key-resignation-certificate 
+                                                                                               ( --cc-cold-key STRING
+                                                                                               | --cc-cold-key-file FILE
+                                                                                               | --cc-cold-key-hash STRING
+                                                                                               )
+                                                                                               --out-file FILE
+
+  Create cold key resignation certificate for a Constitutional Committee Member
+
+Available options:
+  --cc-cold-key STRING     Constitutional Committee cold key (hex-encoded).
+  --cc-cold-key-file FILE  Filepath of the Consitutional Committee cold key.
```

Comment:
We should stick to the current cli format: `--cold-verification-key-file`

Reply:
Done!